lib/OpenQA/WebAPI/Controller/Session.pm

```diff
@@ -16,57 +16,29 @@
 package OpenQA::WebAPI::Controller::Session;
 use Mojo::Base 'Mojolicious::Controller';
 
-use Carp;
-use Net::OpenID::Consumer;
-use URI::Escape;
-use LWP::UserAgent;
-use OpenQA::Schema::Result::Users;
 
 sub ensure_user {
     my ($self) = @_;
-
-    if (!$self->current_user) {
-        $self->redirect_to('login');
-    }
-    else {
-        return 1;
-    }
+    return 1 if $self->current_user;
+    $self->redirect_to('login');
     return;
 }
 
 sub ensure_operator {
     my ($self) = @_;
-
-    if ($self->current_user) {
-        if ($self->is_operator) {
-            return 1 if $self->req->method eq 'GET' || $self->valid_csrf;
-            $self->render(text => 'Bad CSRF token!', status => 403);
-        }
-        else {
-            $self->render(text => "Forbidden", status => 403);
-        }
-    }
-    else {
-        $self->redirect_to('login');
-    }
+    $self->redirect_to('login') and return unless $self->current_user;
```

Comment:
I thought we agreed that `return $self->redirect_to('login') unless $self->current_user` looks better? Old habit?